ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -333,14 +333,14 @@ resources:
 EOF
 
 gcloud deployment-manager deployments create "${INFRA_ID}-bootstrap" --config 04_bootstrap.yaml
-
+BOOTSTRAP_INSTANCE_GROUP=$(gcloud compute instance-groups list --filter "network:${CLUSTER_NETWORK}" | sed "s/ /\n/g" | grep bootstrap)
```

Comment:
Let's use the features of gcloud to return exactly what we want.
Adding `name~^${INFRA_ID}-bootstrap-` to the filter will return only the one instance group that we are looking for. We could probably get rid of the network filter, but it doesn't hurt to have it.
Adding `--format "value(name)"` will have the output include only the name of the instance group.
```suggestion
BOOTSTRAP_INSTANCE_GROUP=$(gcloud compute instance-groups list --filter "network:${CLUSTER_NETWORK} AND name~^${INFRA_ID}-bootstrap-" --format "value(name)"
```